### PR TITLE
[v2.6] k8s july patches

### PR DESCRIPTION
**Pull request by vardhaman22**
for issue: https://github.com/rancher/rancher/issues/42010

### CODE REVIEW

File: rke/k8s_rke_system_images.go [hunk 1]
```diff
@@ -8902,6 +8902,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		// Enabled with v2.6.14
```
Comment (snasovich) [inline rke/k8s_rke_system_images.go]: This will be OOB after v2.6.13.
```suggestion
		// Enabled out of band post v2.6.13
```

**Review by snasovich — DISMISSED** [review]
LGTM, confirmed the only thing that changed is the versions on Kubernetes image.
One nit regarding comment which version it will be enabled in.
Feel free to consider my approval "transferrable" if addressing that one nit is the only difference in future update to this PR.

**Review by jiaqiluo — DISMISSED** [review]
Please address Sergey's comment. Otherwise LGTM

File: rke/k8s_rke_system_images.go (hunk 1 above)
Comment (vardhaman22) [reply rke/k8s_rke_system_images.go]: done

**Review by snasovich — APPROVED** [review]

**Review by jiaqiluo — APPROVED** [review]

**Pull request merged**